blueocean-dashboard/src/main/js/components/PipelineRowItem.jsx

```diff
@@ -7,14 +7,16 @@ import { capable, UrlConfig } from '@jenkins-cd/blueocean-core-js';
 import { MATRIX_PIPELINE } from '../Capabilities';
 import { Icon } from '@jenkins-cd/react-material-icons';
 
-function generateRedirectLink(pipeline) {
+function generateRedirectLink(pipeline, organization) {
     if (capable(pipeline, MATRIX_PIPELINE)) {
+        const fullDisplayPath = organization ? `${organization}/${pipeline.fullDisplayName}` : pipeline.fullDisplayName;
```

Comment:
It's for consistency with line 73 (https://github.com/jenkinsci/blueocean-plugin/pull/1054/files#diff-739e1611c7dd82ec706a63ac6c6fcbfcR73) isn't that the same?